docs/UserGuide.md

```diff
@@ -203,17 +203,19 @@ Lists all tasks in the task list. Clears any existing filters.
 
 Tasks can be filtered by completion status and/or tags. Show only completed tasks with `done/`, pending tasks with `undone/` and tasks with a certain tag `TAG` with `tag/TAG`.
 
+> Note: `undone/` and `done/` flags cannot be specified at the same time.
+
 **Format:**
 
-`task list [done/] [undone/] [t/TAG]`
+`task list [done/ OR undone/] [t/TAG]...`
 
 ### Editing a task: `task edit`
 
 Edits an existing task in the task list
 
 **Format:**
 
-`task edit INDEX [ti/TITLE] [d/DESCRIPTION] [ts/TIMESTAMP] [t/TAG] [c/CONTACT]…`
```

Comment:
The `Features` section of the UG states:

"- Items with `…`​ after them can be used multiple times including zero times."

Probably would want to change that to "Items with `...`" also